diff -urN 4.0.0beta2.orig/frontends/php/include/classes/core/CConfigFile.php 4.0.0beta2/frontends/php/include/classes/core/CConfigFile.php
--- 4.0.0beta2.orig/frontends/php/include/classes/core/CConfigFile.php	2018-09-10 16:56:00.508027723 +0300
+++ 4.0.0beta2/frontends/php/include/classes/core/CConfigFile.php	2018-09-10 16:55:01.253198416 +0300
@@ -24,7 +24,7 @@
 	const CONFIG_NOT_FOUND = 1;
 	const CONFIG_ERROR = 2;
 
-	const CONFIG_FILE_PATH = '/conf/zabbix.conf.php';
+	const CONFIG_FILE_PATH = '/etc/zabbix/web/zabbix.conf.php';
 	const BRAND_CONFIG_FILE_PATH = '/conf/brand.conf.php';
 
 	private static $supported_db_types = [
diff -urN 4.0.0beta2.orig/frontends/php/include/classes/core/ZBase.php 4.0.0beta2/frontends/php/include/classes/core/ZBase.php
--- 4.0.0beta2.orig/frontends/php/include/classes/core/ZBase.php	2018-09-10 16:56:00.508027723 +0300
+++ 4.0.0beta2/frontends/php/include/classes/core/ZBase.php	2018-09-10 16:54:04.046330303 +0300
@@ -272,7 +272,7 @@
 	 * @throws Exception
 	 */
 	protected function setMaintenanceMode() {
-		require_once $this->getRootDir().'/conf/maintenance.inc.php';
+		require_once '/etc/zabbix/web/maintenance.inc.php';
 
 		if (defined('ZBX_DENY_GUI_ACCESS')) {
 			$user_ip = (isset($_SERVER['HTTP_X_FORWARDED_FOR']) && !empty($_SERVER['HTTP_X_FORWARDED_FOR']))
@@ -288,7 +288,7 @@
 	 * Load zabbix config file.
 	 */
 	protected function loadConfigFile() {
-		$configFile = $this->getRootDir().CConfigFile::CONFIG_FILE_PATH;
+		$configFile = CConfigFile::CONFIG_FILE_PATH;
 		$config = new CConfigFile($configFile);
 		$this->config = $config->load();
 	}
diff -urN 4.0.0beta2.orig/frontends/php/include/classes/setup/CSetupWizard.php 4.0.0beta2/frontends/php/include/classes/setup/CSetupWizard.php
--- 4.0.0beta2.orig/frontends/php/include/classes/setup/CSetupWizard.php	2018-09-10 16:56:00.512027645 +0300
+++ 4.0.0beta2/frontends/php/include/classes/setup/CSetupWizard.php	2018-09-10 16:54:04.046330303 +0300
@@ -333,7 +333,7 @@
 	function stage5() {
 		$this->setConfig('ZBX_CONFIG_FILE_CORRECT', true);
 
-		$config_file_name = Z::getInstance()->getRootDir().CConfigFile::CONFIG_FILE_PATH;
+		$config_file_name = CConfigFile::CONFIG_FILE_PATH;
 		$config = new CConfigFile($config_file_name);
 		$config->config = [
 			'DB' => [
@@ -502,7 +502,7 @@
 				// make zabbix.conf.php downloadable
 				header('Content-Type: application/x-httpd-php');
 				header('Content-Disposition: attachment; filename="'.basename(CConfigFile::CONFIG_FILE_PATH).'"');
-				$config = new CConfigFile(Z::getInstance()->getRootDir().CConfigFile::CONFIG_FILE_PATH);
+				$config = new CConfigFile(CConfigFile::CONFIG_FILE_PATH);
 				$config->config = [
 					'DB' => [
 						'TYPE' => $this->getConfig('DB_TYPE'),
